src/_guides/bump-sh-tutorials/feedback.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Improve Your API with User Feedback
+authors: phil
+excerpt: Make your API better by listening to the people who use it probably more than you do.
+date: 2024-10-31
+---
+
+Gathering user feedback is essential to improving your API and ensuring it meets users' needs. With Bump.sh, you can integrate feedback forms directly into your OpenAPI documentation by embedding a link to a form using [`x-feedbackLink` extension](/help/publish-documentation/feedback/). 
+
+This guide will show you how to set up a feedback form using Google Forms or Typeform and embed it seamlessly into your API documentation.
+
+## Step 1: Create Your Feedback Form
+
+First, choose your preferred form tool—either Google Forms or Typeform. Both options are excellent, but here’s a quick overview:
+
+- **Google Forms** is free, simple, and perfect for straightforward surveys.
+- **Typeform** offers a more visually engaging experience and may work better if you want an interactive form with a modern look.
+- **Notion** has a new Form builder, and seeing as loads of people already run their whole life through Notion, this might be the easiest option for many.
+
+Once you’ve chosen a tool:
+
+- **For Google Forms**: Go to [Google Forms](https://forms.google.com), and either start from a blank form or choose a template. Create questions to capture feedback on your API, like user satisfaction, feature requests, or any issues they’ve encountered.
+- **For Typeform**: Head to [Typeform](https://typeform.com), log in, and click "Create a typeform." Set up your questions, keeping them focused on the feedback you need to improve your API.
+- **For Notion**: Go to [Notion](https://www.notion.so), and once you have a workspace, add a New Page somewhere, and look for the "Form" option under "Getting started..."
+
+The tools all have a different form building, but the Notion builder should look like this.
+
+![A few text boxes with questions asking "What are we doing well?", "What would you like to see improved?", and "Email"](/images/guides/feedback/notion-form-builder.png)
+
+- **In Google Forms**: Click the “Send” button, select the link icon, and copy the link (something like `https://forms.gle/XYZ`).
+- **In Typeform**: Click "Share" at the top, then copy the link (typically `https://yourformname.typeform.com/to/XYZ`).
+- **In Notion**: Click "Share Form", then copy the link (`https://www.notion.so/bumpsh/XYZ`)
+
+![The "Share form" button has been activated, opening a URL box with a "Copy link" next to it](/images/guides/feedback/notion-send-form.png)
```

Comment:
@philsturgeon looks like an image is missing 😄

Reply:
Fixed